.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+defaults: &defaults
+  working_directory: ~
+  docker:
+  - image: debian:stretch
+
+version: 2
+jobs:
+  build:
+    <<: *defaults
+    steps:
+    - run:
+        name: Install system dependencies
+        command: |
+          apt-get update
+          apt-get install build-essential curl dh-autoreconf git libpq-dev postgresql postgresql-client -y
+          apt-get --purge autoremove
+          apt-get clean
+    - checkout
+    - run:
+        name: Install the Haskell tool stack
+        command: |
+          curl -sSL https://get.haskellstack.org/ | sh
+    - restore_cache:
+        keys:
+        - v2-{{ checksum "stack.yaml" }}
+    - run:
+        name: stack setup
+        command: |
+          stack setup
+    - run:
+        name: stack build
+        command: |
+          stack build
+    - run:
```

Comment:
let's remove `stack install` since we don't need the binaries copied to `~/.local/bin` for anything.

The [stack docs](https://docs.haskellstack.org/en/stable/build_command/#synonyms) explain that `stack install` is just an alias for `stack build --copy-bins`.